plugins/modules/helm_cli.py

```diff
@@ -0,0 +1,593 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: helm_cli
+short_description: Manages Kubernetes packages with the Helm package manager
+description:
+  - Install, upgrade, delete packages with the Helm package manager.
+version_added: "2.10"
+author:
+  - Lucas Boisserie (@LucasBoisserie)
+  - Matthieu Diehr (@d-matt)
+requirements:
+  - "helm (https://github.com/helm/helm/releases)"
+  - "yaml (https://pypi.org/project/PyYAML/)"
+options:
+  binary_path:
+    description:
+      - The path of a helm binary to use.
+    required: false
+    type: path
+  chart_ref:
+    description:
+      - chart_reference on chart repository
```

Comment:
```suggestion
      - chart_reference on chart repository.
```

Full stop at the end of each line in the description.

Reply:
Updated in last commit